test/support/fixtures/dynamic/invalid_input_types.exs

```diff
@@ -1,19 +1,16 @@
-defmodule Absinthe.Fixtures.InvalidInputTypesSchema do
```

Comment:
Don't you have the file names the other way around now? InvalidOutputTypesSchema is in the invalid_input_types.exs file and the other way around.